Update Spinnaker templates to use Halyard

ejizba opened the pull request:
1. Simplify the config by using Halyard
2. Deploy spinnaker _to_ the k8s cluster rather than the VM
3. Deprecate the 201 template

clguiman commented on spinnaker/install_halyard/install_halyard.sh:
```diff
@@ -0,0 +1,59 @@
+throw_if_empty username $username
+
+# Install Halyard
+curl --silent -O https://raw.githubusercontent.com/spinnaker/halyard/master/install/stable/InstallHalyard.sh
```
why don't you pipe this script to bash?

ejizba replied:
I don't remember :) fixed

ejizba committed:
Update Spinnaker templates to use Halyard

1. Simplify the config by using Halyard
2. Deploy spinnaker _to_ the k8s cluster rather than the VM
3. Deprecate the 201 template
